wsd/DocumentBroker.cpp

```diff
@@ -1975,6 +1980,12 @@ size_t DocumentBroker::removeSession(const std::string& id)
         // If last editable, save and don't remove until after uploading to storage.
         if (!lastEditableSession || !autoSave(isPossiblyModified(), dontSaveIfUnmodified))
```

Comment:
I worry about the last readonly session - how do we mark to destroy for those sessions ?

Reply:
Good point. Improved.